README.md

```diff
@@ -109,7 +110,13 @@ There are a couple of options that can be tweaked:
   active. If for some reason loading a snapshot takes a while (e.g. stored on
   remote storage), setting this to `nil` will make moving around in the
   timeline more responsive.
+* `snapshot-timemachine-snapshot-traceback` (default: 14) determines how far
+  back to search for distinct snapshots. Currently only supported for ZFS.
+* `snapshot-timemachine-snapshot-finder` (default: `snapshot-timemachine-snapper-snapshot-finder`) function used to retrieve the snapshots for a givn file. Current options are
+  * snapshot-timemachine-snapper-snapshot-finder: Uses [Snapper] to discover past snapshots
+  * snapshot-timemachine-zsd-snapshot-finder: Uses [zsd](zfs-snap-diff) to discover past snapshots
 
+[zsd]: https://github.com/j-keck/zfs-snap-diff
```

Comment:
`zsd` is now a independent project and lives here:  https://github.com/j-keck/zsd
and the website is: https://j-keck.github.io/zsd